src/package-copy.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "",
+    "version": "0.0.0",
+    "description": "",
```

I think we can prefill these (most cli generators do). Something like:
```suggestion
    "name": "tsprogen-project",
    "version": "0.0.0",
    "description": "Boilerplate generated by TSProGen.",
```